main.cpp

```diff
@@ -1,44 +1,22 @@
 #include <SFML/Graphics.hpp>
-
 #include <iostream>
 
-#include "Player.hpp"
-#include "Animation.hpp"
+#include "Objects/Component.hpp"
+#include "Objects/GameObject.hpp"
 
 #define SCREEN_WIDTH 1024
 #define SCREEN_HEIGHT 768
 #define GAME_TITLE "Run&Dash"
 
-int main()
-{
+
+int main() {
+
 	//Tworzenie okna glownego GUI
 	sf::RenderWindow window(sf::VideoMode(SCREEN_WIDTH, SCREEN_HEIGHT), GAME_TITLE);
 	window.setFramerateLimit(60);	//Limit klatek do 60
 
-	Player player = Player(0, 0);	//Stworzenie obiektu gracza na pozycji (10,20)
-	player.setPosition( 0, SCREEN_HEIGHT-player.spriteHeight*player.scale );
-	
-	Animation animationIdle = Animation(&player.shape);	//Dodanie animacji do gracza
-	Animation animationRunning = Animation(&player.shape);
-
-	TFrame framesIdle[4] = {
-		{sf::IntRect(0*player.spriteWidth,0,player.spriteWidth,player.spriteHeight), 0.15f},
-		{sf::IntRect(1*player.spriteWidth,0,player.spriteWidth,player.spriteHeight), 0.15f},
-		{sf::IntRect(2*player.spriteWidth,0,player.spriteWidth,player.spriteHeight), 0.15f},
-		{sf::IntRect(3*player.spriteWidth,0,player.spriteWidth,player.spriteHeight), 0.2f}
-	};
-
-	TFrame framesRunning[6] = {	
-		{sf::IntRect(1*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-		{sf::IntRect(2*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-		{sf::IntRect(3*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-		{sf::IntRect(4*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-		{sf::IntRect(5*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-		{sf::IntRect(6*player.spriteWidth,1*player.spriteHeight, player.spriteWidth, player.spriteHeight), 0.15f},
-	};
-	
-	animationIdle.addFrames(framesIdle, 4);
-	animationRunning.addFrames(framesRunning, 6);	
+	sf::RectangleShape* rec = new sf::RectangleShape(sf::Vector2f(0.0f, 0.0f));
```

Comment:
Tak btw, ten prostokąt ma wymiary 0x0 :p

Reply:
A to było tak żeby przetestować czy się buduje.